<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTViewer</class>
 <widget class="QMainWindow" name="CTViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>939</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>11</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CT Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="2">
     <widget class="QVTKWidget" name="qvtkWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="grpFiles">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Shown planes</string>
         </property>
         <widget class="QCheckBox" name="showSagital">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>70</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Sagittal</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="showCoronal">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>70</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Coronal</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="showAxial">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>91</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Axial</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="showCustom">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>70</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Active plane</string>
         </property>
         <widget class="QRadioButton" name="btnSagittalActive">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Sagittal</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="btnCoronalActive">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Coronal</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="btnAxialActive">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Axial</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="btnCustomActive">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pbOpenDicom">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Open DICOM directory</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>130</height>
          </size>
         </property>
         <property name="title">
          <string>Custom plane</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>20</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>46</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Origin</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>20</y>
            <width>46</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Point 1</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>20</y>
            <width>46</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Point 2</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="tbOx">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="tbOy">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>70</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="tbOz">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>100</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="tbP1x">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>40</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="tbP1y">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>70</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="tbP1z">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>100</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="tbP2x">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>40</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="tbP2y">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>70</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="tbP2z">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>100</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btnRefreshCustom">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>100</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Set coords.</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTextBrowser" name="tbrowseLog">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
